package.json

```diff
@@ -97,7 +105,7 @@
     "react-invenio-forms": "^0.10.7",
     "react-overridable": "^0.0.3",
     "react-redux": "^7.2.0",
-    "react-scripts": "^3.4.0",
+    "react-scripts": "^5.0.1",
```

Comment:
Not sure about this changes, we would need to check breaking changes and maybe upgrade it in all other repos like [react-invenio-forms](https://github.com/inveniosoftware/react-invenio-forms/blob/master/package.json#L66). Are you sure you need all this changes in the `package.json`?

Reply:
These changes were made because otherwise eslint was failing to lint on my system/editor. Because they are dev dependencies my assumption is that they are only used for development or when building the library, that should make it safe to upgrade (we have some quite old dependencies). What do you think?

Reply:
This should not be a problem, because they are `devDependencies`: when packaged and released on `npmjs.org`, the modules using this one will not install these dependencies.
What we need always to pay attention is `dependencies` and `peerDeps`.